docs/source/release/v6.13.0/reflectometry.rst

```diff
@@ -2,6 +2,11 @@
 Reflectometry Changes
 =====================
 
+.. image:: ../../images/6_13_release/refl-gui-fredrikze.png
+   :class: screenshot
+   :width: 45%
+   :align: right
```

Comment:
It's not clear what the image is illustrating. Also - can we get a screenshot that is more typical of what users will see? I'm not sure how many use dark mode with hot pink accents.

Reply:
Fair point, this is just how my workbench has always looked